.rubocop.yml

```diff
@@ -158,6 +159,12 @@ Lint/ModuleDisclosureDatePresent:
     # Only exploits require disclosure dates, but they can be present in auxiliary modules etc.
     - 'modules/exploits/**/*'
 
+Lint/SideEffectsInNotes:
+  Include:
+    # Only exploits and auxiliary modules require SideEffects to be listed.
+    - 'modules/exploits/**/*'
+    - 'modules/auxiliary/**/*'
```

Comment:
Let's add `modules/post` as well.

And per @adfoster-r7 's idea we could expand this to include `Stability` and `Reliability` keys.